src/routes/transactions/mappers/module-transactions/module-transaction.mapper.ts

```diff
@@ -0,0 +1,43 @@
+import { AddressInfoHelper } from '../../../common/address-info/address-info.helper';
+import { Injectable } from '@nestjs/common';
+import { Safe } from '../../../../domain/safe/entities/safe.entity';
+import { Transaction } from '../../entities/transaction.entity';
+import { MultisigTransactionInfoMapper } from '../common/transaction-info.mapper';
+import { ModuleTransactionStatusMapper } from './module-transaction-status.mapper';
+import { ModuleTransaction } from '../../../../domain/safe/entities/module-transaction.entity';
+import { ModuleExecutionInfo } from '../../entities/module-execution-info.entity';
+
+@Injectable()
+export class ModuleTransactionMapper {
+  constructor(
+    private readonly addressInfoHelper: AddressInfoHelper,
+    private readonly statusMapper: ModuleTransactionStatusMapper,
+    private readonly transactionInfoMapper: MultisigTransactionInfoMapper,
+  ) {}
+
+  async mapTransaction(
+    chainId: string,
+    transaction: ModuleTransaction,
+    safe: Safe,
+  ): Promise<Transaction> {
+    const txStatus = this.statusMapper.mapTransactionStatus(transaction);
+    const txInfo = await this.transactionInfoMapper.mapTransactionInfo(
+      chainId,
+      transaction,
+      safe,
+    );
+    //TODO executionInfo = contractInfo(transaction.module)
```

Comment:
Is this TODO comment correct? (I think `executionInfo` is built in the next line)

Reply:
I forgot to remove it.

Reply:
Done https://github.com/5afe/safe-client-gateway-nest/pull/205/commits/742120c589ab19309cace7525f86565b96b74c16